### ISSUE TITLE
Check integration of python package BayesPowerlaw into R

### ISSUE THREAD

**Issue by arunge** [Pandora-IsoMemo/drat#43]
... feedback on some of the apps from external collaborators is quite slow. I was thinking of replacing some of the apps for now. To start with, can we replace ABM by this Python app: https://github.com/AtwalLab/BayesPowerlaw

_Originally posted by @isomemo in https://github.com/Pandora-IsoMemo/drat/issues/38#issuecomment-2325990638_

**Comment by arunge** [comment]
- [ ] check if integration with reticulate into R is working without erros
- [ ] check if setting up a docker container with a minimal shiny app that contains the installation is working without errors

**Comment by arunge** [comment]
@isomemo If I remember correctly, you mentioned that the plan is to
- postpone the development of [deBInferS](https://github.com/Pandora-IsoMemo/deBInferS) and
- instead check the python package mentioned above

Correct? Let me know, if you would like another of the new apps to be postponed.

**Comment by f-lukas** [comment]
I have installed the package [BayesPowerlaw](https://github.com/AtwalLab/BayesPowerlaw) with pip as mentioned in the [package documentation](https://bayespowerlaw.readthedocs.io/en/latest/). I then tried to import the package with python which already failed. I also tried importing the package in R with reticulate which also failed with the same error message.

![Bildschirmfoto vom 2024-09-27 14-07-15](https://github.com/user-attachments/assets/dab623dd-2134-42b8-9af9-ef8c5a7c73a9)

```
 >>> import BayesPowerlaw as bp
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "./BayesPowerlaw/__init__.py", line 1, in <module>
    from BayesPowerlaw.src.BayesPowerlaw import bayes
ModuleNotFoundError: No module named 'BayesPowerlaw.src'
```
![Bildschirmfoto vom 2024-09-27 14-11-49](https://github.com/user-attachments/assets/6f03236d-4408-4735-a6be-a1ce1e8960a1)


The package does not seem to have a correct package structure. The last update of the package was 5 years ago so the structure might be outdated. We should discuss in the next meeting if and how we should continue here.
